### PR TITLE
Heal 31, Heal 32, Heal 75, Heal 77

### PR DESCRIPTION
**Pull request by aishwaryaravi19**
Screenshot 2023-04-11 at 11 05 42 PM
Screenshot 2023-04-09 at 7 14 58 PM
Screenshot 2023-04-11 at 11 05 53 PM

### CODE REVIEW

File: client/src/components/Doctor/DoctorDashboard.tsx
Comment (varunalla) [inline client/src/components/Doctor/DoctorDashboard.tsx]: remove if unnecessary. The file upload functionality is not implemented

File: client/src/components/Doctor/DoctorDashboard.tsx
Comment (varunalla) [inline client/src/components/Doctor/DoctorDashboard.tsx]: encrypt files being uploaded to s3 as doctor credentials are confidential

File: client/src/components/HealthCareProvider/HCProviderDashboard.tsx
Comment (varunalla) [inline client/src/components/HealthCareProvider/HCProviderDashboard.tsx]: if request.approved is false, there is no distinction between pending and rejected requests. this seems to be a bug.

File: client/src/providers/HealthProvider.tsx
Comment (varunalla) [inline client/src/providers/HealthProvider.tsx]: if request.approved is false, there is no distinction between pending and rejected requests. this seems to be a bug. how will the request be associated with health care provider?

File: client/src/providers/HealthProvider.tsx
Comment (varunalla) [inline client/src/providers/HealthProvider.tsx]: install prettier and make sure this is formatted properly

File: client/src/providers/HealthProvider.tsx
Comment (varunalla) [inline client/src/providers/HealthProvider.tsx]: please remove all console.log in which ever section you see them.

File: client/src/providers/HealthProvider.tsx
Comment (varunalla) [inline client/src/providers/HealthProvider.tsx]: is this call necessary? can you explain why?

File: config/constants.js
Comment (varunalla) [inline config/constants.js:4]: remove console.log and fix formatting

**Review by varunalla — CHANGES_REQUESTED** [review]
Please delete contents of client/src/contracts folder. commit it and run type check and  checkin those files.

File: client/src/providers/HealthProvider.tsx
Comment (aishwaryaravi19) [reply client/src/providers/HealthProvider.tsx]: yes, it is necessary to set verification request after each approval/rejection

File: client/src/components/Doctor/DoctorDashboard.tsx
Comment (aishwaryaravi19) [reply client/src/components/Doctor/DoctorDashboard.tsx]: It will be addressed as part of HEAL 83 & 84

**Review by varunalla — APPROVED** [review]
approved! please test once merged!

**Pull request merged**
